### PR TITLE
Quest stuff 11/28

### PR DESCRIPTION
**Pull request by King65868**

### CODE REVIEW

**Review by WaitingIdly — CHANGES_REQUESTED** [review]

File: overrides/config/betterquesting/DefaultQuests.json [hunk 1]
```diff
@@ -21103,7 +21103,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Part of the Golden Club",
-          "desc:8": "Craft a Golden Club! You will need more than one, although likely not that many. If used optimally, you won\u0027t need more than 2.\n\nBy swinging this in the air it uses up 1 durability to summon Lightning! If you want to use it for anything useful (other than looks), then make sure to use it WHILE HOVERING OVER a block. This is required for both Lightning transformations and LE (Lightning Energy) generation."
+          "desc:8": "Craft a Golden Club! You only need one, but you can make more if you want.\n\nBy swinging this in the air it uses up 1 durability to summon Lightning! If you want to use it for anything useful (other than looks), then make sure to use it WHILE HOVERING OVER a block. This is required for LE (Lightning Energy) generation but Lightning transformations can be done by other sources of lightning as well."
```
[the remark this answers is not in the document]
Comment (WaitingIdly) [reply overrides/config/betterquesting/DefaultQuests.json:21106]: you made them all uppercase. the summoned object "lightning" is not a proper name, it should be lowercase.

File: overrides/config/betterquesting/DefaultQuests.json [hunk 2]
```diff
@@ -78480,7 +78480,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Ultimate Energy Storage",
-          "desc:8": "Craft an Ultimate Energy Cube! For now, this will be your best way of storing energy! ...But wait until you get into EnderIO or Draconic."
+          "desc:8": "Craft an Ultimate Energy Cube! For now, this will be your best way of storing energy! ...But wait until you get into late Mekanism or Draconic."
```
[the remark this answers is not in the document]
Comment (WaitingIdly) [reply overrides/config/betterquesting/DefaultQuests.json:78483]: not asking to revert, just asking to keep EnderIO.

File: overrides/config/betterquesting/DefaultQuests.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (King65868) [reply overrides/config/betterquesting/DefaultQuests.json:21106]: Is that good?

**Comment by King65868** [comment]
This should be good now

**Review by WaitingIdly — APPROVED** [review]

File: overrides/config/betterquesting/DefaultQuests.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (WaitingIdly) [reply overrides/config/betterquesting/DefaultQuests.json:78483]: whoops, missed this: `EnderIO`, not `Enderio`

**Pull request merged**